intelmq/lib/test.py

```diff
@@ -46,7 +46,7 @@
               }
 
 
-class Parameters:
+class Parameters(object):
```

Comment:
```suggestion
class Parameters():
```
Adding object as parent is not necessary on Python 3 any more

Reply:
I don't believe I modified intelmq/lib/test.py - this must be a synchronization issue